src/ebfloeseg/app.py

```diff
@@ -7,9 +7,6 @@
 import typer
 from typing import Optional
 
-
-import pandas as pd
-
```

Comment:
This removal will need to be removed when you solve the merge conflicts – pandas is still used in app.py, but by other functions now.